package/network/utils/layerscape/restool/patches/010-gcc11.patch

```diff
@@ -0,0 +1,11 @@
+--- a/restool.h
++++ b/restool.h
+@@ -60,7 +60,7 @@
+ /**
+  * Max length of a device file
+  */
+-#define DEV_FILE_SIZE		15
++#define DEV_FILE_SIZE		16
```

Comment:
This is also part of this upstream patch:
https://source.codeaurora.org/external/qoriq/qoriq-components/restool/commit/?h=la1224rdb-early-access&id=91b82e3db29b694e4445c23e5930b379064e884b

This also fixes some other problems with gcc 10.2

Reply:
Doesn't apply cleanly. Maybe it's better to update restool...